src/lib/icons/IconThumbDownCircle.svelte

```diff
@@ -0,0 +1,29 @@
+<!-- source: DFINITY foundation -->
+<svg
+  width="48"
```

Comment:
Let's use the same pattern from other icons where the size is a prop:
https://github.com/dfinity/gix-components/blob/472762006c2753d1e79902b4841c9c865c3b2f5c/src/lib/icons/IconAccountBalance.svelte#L5